### PR TITLE
Adding flag to conditionally disable iptables_manager job in services

### PR DESCRIPTION
**Pull request by jayeshjk**

### CODE REVIEW

File: data-access-layer/bosh/manifest/Addons.js [hunk 1]
```diff
@@ -24,6 +25,15 @@ class Addons {
     }
   }
 
+  shouldEnableConnectivity() {
+    const connectivityAllowedServicesList = _.get(this.context, 'connectivityAllowedServicesList');
```
Comment (subhankarc) [inline data-access-layer/bosh/manifest/Addons.js:29]: Is this necessary ? If this is for test, then can we not emulate with the params rather ?

Comment (jayeshjk) [reply data-access-layer/bosh/manifest/Addons.js:29]: My understanding was that we need to do this for only postgresql, hence this extra check. WDYT from that perspective?

Comment (subhankarc) [reply data-access-layer/bosh/manifest/Addons.js:29]: yeah...makes total sense not to open up for all. 🥇

File: operators/bosh-operator/DirectorService.js [hunk 2]
```diff
@@ -953,6 +953,8 @@ class DirectorService extends BaseDirectorService {
   }
 
   configureAddOnJobs(serviceManifest, context) {
+    _.set(context, 'serviceId', this.service.id);
+    _.set(context, 'connectivityAllowedServicesList', ['6db542eb-8187-4afc-8a85-e08b4a3cc24e']);
```
Comment (subhankarc) [inline operators/bosh-operator/DirectorService.js:957]: Can we remove this hardcoded part then ?

Comment (jayeshjk) [reply operators/bosh-operator/DirectorService.js:957]: sure, I will modify to accept this as part of service catalog attribute, and will enable for postgresql for now.

**Pull request merged**